mne_features/feature_extraction.py

```diff
@@ -185,34 +185,60 @@ def _apply_extractor(extractor, X, return_as_df):
     return X, feature_names
 
 
-def _check_func_names(selected, feature_funcs_names):
-    """ Checks if the names of selected feature functions match the available
-    feature functions.
+def _check_funcs(selected, feature_funcs):
+    """ Checks if the elements of ``selected`` are either strings (alias of a
+    feature function defined in mne-features) or tuples of the form
+    ``(str, callable)`` (user-defined feature function).
```

Comment:
please check pep257 for your docstrings:

https://www.python.org/dev/peps/pep-0257/

there is even a tool to automate testing: https://pypi.python.org/pypi/pep257